cosmos/icplaza_9000.json

```diff
@@ -0,0 +1,55 @@
+{
+  "rpc": "http://icplazarpc.ic-plaza.org",
+  "rest": "http://icplazarest.ic-plaza.org",
+  "chainId": "icplaza_9000-1",
+  "chainName": "ICPlaza",
+  "nodeProvider": {
+    "name": "ICPlaza Node",
+    "email": "0xf48892@gmail.com",
+    "website":"https://www.ic-plaza.org/"
+  },
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/icplaza_9000/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "ICT",
+    "coinMinimalDenom": "uict",
+    "coinDecimals": 18,
+    "coinGeckoId": "ict"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/icplaza",
```

Comment:
@chuanfoo Could you remove `walletUrl` field?

Reply:
ok